docker-compose.integration.yml

```diff
@@ -34,7 +34,7 @@ services:
       - "5211:80"
       
   blob.store:
-    image: mcr.microsoft.com/azure-storage/azurite
+    image: mcr.microsoft.com/azure-storage/azurite:3.10.0
```

Comment:
Why is this version fixed?

Reply:
The default/latest image isn't actually the latest. It's something like 3.8. The health checks default to the latest version of the storage API but 3.8 doesn't support the latest version of the storage API so the health checks fail.

It would be better if Microsoft would update the latest tag to point to 3.10 but they haven't done so yet, which means for now the version needs to be fixed 3.10. However, I'll create a tech debt ticket to keep an eye on the tags and come back to revert this to latest once the tags are updated.